analyses/doublet-detection/run_doublet-detection-scpca.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script runs doublet detection over ScPCA data for a given project
+# Usage: ./run_doublet-detection-scpca.sh {scpca project id}
+
+set -euo pipefail
+
+
+# Ensure script is being run from its directory
+MODULE_DIR=$(dirname "${BASH_SOURCE[0]}")
+cd ${MODULE_DIR}
+
+PROJECT_ID=$1
+
+# Define directories
+DATA_DIR="../../data/current"
+RESULTS_DIR="results/scpca-results/${PROJECT_ID}"
+mkdir -p ${RESULTS_DIR}
+
+# Detect doublets on each processed SCE file in each sample directory
+for SAMPLE_DIR in ${DATA_DIR}/${PROJECT_ID}/SCPCS*; do
+    SAMPLE_BASE=$(basename $SAMPLE_DIR)
```

Comment:
```suggestion
    SAMPLE_ID=$(basename $SAMPLE_DIR)
```
This just returns the sample ID correct? Let's be explicit in how we name that then.